projects/simulation/human_model_generation/demos/model_generation.ipynb

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
It would be good to add a short README to explain how to run the jupyter notebook of the demo, what to install etc (or at least to point to where this information can be found).